src/client/app/reducers/readings.js

```diff
@@ -16,7 +16,12 @@ import * as readingsActions from '../actions/readings';
  * @type {State~Readings}
  */
 const defaultState = {
```

Comment:
That's a good point, however, what do we do about actions like fetchAllNeededReadings()? If we have two files: actions/barReadings.js and actions/lineReadings.js, it doesn't seem to be appropriate to place it in either one.